### ISSUE TITLE
Expected result from the reranking without indexing

### ISSUE THREAD

**Issue by mnskim** [stanford-futuredata/ColBERT#8]
Hi Omar,
Is there some number we should be looking for, roughly speaking, when doing the reranking (without indexing) on the msmarco dev set? (EDIT: Would be equivalent to Table 1 in the paper?)
The reason is that I'd like to be able to use this performance to gauge the performance of the full indexed retrieval. Would this work, or does this metric not really correlate with the full indexed performance?

I trained a model with a slightly larger batch size and without gradient accumulation for 200000 steps, and using the msmarco evaluation script from anserini, the MRR@10 is around 0.3514. Does this look reasonable?

**Comment by okhat** [comment]
Hi Minsoo,

Yes, Table 1 has re-ranking results. And, yes, this is a good way to validate the model before indexing. (Notice that, in general, you could re-rank other top-k sets too, not just the official top-1000. Some could give you a better idea of the final model score than others.)

And 35.14% MRR@10 is pretty good for re-ranking.

**Comment by mnskim** [comment]
Hi Omar, 
Thank you for the fast response! Reranking other topk sets sounds like a great idea, I will try that as well. Thanks!!
